.travis/elasticsearch.yml

```diff
@@ -0,0 +1,5 @@
+cluster.name: "docker-cluster"
```

Comment:
Is the `cluster.name` necessary here?  I think "docker-cluster" may be the default...

Reply:
`cluster.name` is not necessary, by default `cluster.name` is `elasticsearch`, [link to the doc](https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster.name.html#cluster.name).